data/mods/Tanks/items.json

```diff
@@ -422,7 +422,7 @@
   "sight_dispersion": 60,
   "recoil": -20,
   "durability": 8,
-  "clip_size": 1,
+  "clip_size": 5,
   "reload": 100,
   "loudness" : 300,
   "flags": [ "NO_RELOAD" ]
```

Comment:
Tested further. That seems to be incorrect. Even with magazines assigned, currently weapons with reloading disabled will ignore the flag if they are installed on a vehicle. This is fortunate, as I can see no other way to preserve intended behavior.